src/views/compute/InstanceTab.vue

```diff
@@ -354,6 +370,45 @@ export default {
         this.secondaryIPs = response.listnicsresponse.nic[0].secondaryip
       })
     },
+    fetchGuestNetwork (networkid) {
```

Comment:
@utchoang Is this method used anywhere ?

Reply:
@davidjumani No. This is a function I testing. I'll remove it. Thanks.